salt/modules/debbuild.py

```diff
@@ -600,7 +610,7 @@ def make_repo(repodir,
     # preset passphase and interaction with gpg-agent
     gpg_info_file = '{0}/gpg-agent-info-salt'.format(gnupghome)
     gpg_tty_info_file = '{0}/gpg-tty-info-salt'.format(gnupghome)
-    gpg_tty_info_dict = {}
+#    gpg_tty_info_dict = {}
```

Comment:
If this isn't used, let's just remove it entirely.

Reply:
Sorry about that, usually have my commented out with a DGM prefix, and in this case didn't, hence missed in search for cleanup,, usage cleaned up.